README.md

```diff
@@ -51,7 +69,14 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+// var pizza;
+// pizza.temperature = 70;
+// pizza.temperature = temperature
+// function bake() {
+// temperature = 300
+// }
+// return temperature
+// temperature = pizza.temperature
```

Comment:
Why is this all commented out?

The third line, `pizza.temperature = temperature`, would throw an error saying that `temperature` is undefined. You have a `pizza.temperature` variable; you don't have a `temperature` variable.

`return` only goes inside functions.

This asks you to define an object with properties and methods. That should look something like:

```js
var myObject = {
  myProperty: "hello",
  myMethod: function(){
    console.log("Hi there")
  }
}
```